### PR TITLE
Emit error when simple_verify fails.

### PR DESCRIPTION
**Pull request by jsha**
When running the manual authenticator, if simple_verify fails, there is no
output to indicate what went wrong, just "Incomplete authorizations."

### CODE REVIEW

File: letsencrypt/plugins/manual.py [hunk 1]
```diff
@@ -182,6 +182,8 @@ def _perform_single(self, achall):
                 achall.account_key.public_key(), self.config.simple_http_port):
             return response
         else:
+            sys.stdout.write(
+                "Self-verify of challenge failed, authorization abandoned.\n")
```
Comment (kuba) [inline letsencrypt/plugins/manual.py]: can we turn that into `log.error` please?

**Comment by jsha** [comment]
Fixed, thanks!

File: letsencrypt/plugins/manual.py [hunk 2]
```diff
@@ -182,6 +182,8 @@ def _perform_single(self, achall):
                 achall.account_key.public_key(), self.config.simple_http_port):
             return response
         else:
+            logger.error(
+                "Self-verify of challenge failed, authorization abandoned.\n")
```
Comment (kuba) [inline letsencrypt/plugins/manual.py]: `\n` is not necessary in logs

**Comment by kuba** [comment]
LGTM

**Comment by jsha** [comment]
Fixed the newline, thanks.

**Pull request merged**